### PR TITLE
Api 20 read from envvars

### PR DESCRIPTION
**Pull request by jgoldsmi**
https://offerpop.atlassian.net/browse/API-20

### CODE REVIEW

**Comment by grampajoe** [comment]
Looks good to me, very Java. :+1:

I wonder whether it'd be simpler to just copy this code into the `gnip-kinesis-connector` repo instead of using it as an external(ish?) library, but obviously that's outside the scope of this.

I'm also going to ask whether you could write some tests for this, but given that I wouldn't know where to start doing that, it's probably also far outside the scope. :telescope:

**Comment by jgoldsmi** [comment]
Added some tests, and made a ticket to merge this with `gnip-kinesis-connector` (https://offerpop.atlassian.net/browse/API-63).

**Comment by yiwork** [comment]
:+1:  on merging the repos together. The initial point of keeping them separate was so we can post a PR against Gnip's repo, and in event they accepted our PR we can then pull it back down. There is probably no hope of this any more.

**Comment by klivan** [comment]
@yiwork both our pull requests are open and nobody is merging them in https://github.com/gnip/sample-kinesis-connector/pulls

File: src/main/java/com/twitter/kinesis/utils/Environment.java [hunk 1]
```diff
@@ -36,70 +36,69 @@ public void logProperties() {
   }
```
Comment (roviedo) [inline src/main/java/com/twitter/kinesis/utils/Environment.java:36]: we might want to add the replay variables to be loaded from envvars to this ticket too. The replay variables to be loaded are in this pull request https://github.com/offerpop/gnip-kinesis-connector/pull/2. The changes in this pull request need to be added to the sample-kinesis-connector when working on the ticket to add replay support.

Comment (jgoldsmi) [inline src/main/java/com/twitter/kinesis/utils/Environment.java:36]: What's the state of that repo? Should we even have this one if we've copied the code into offerpop/gnip-kinesis-connector ?

**Pull request merged**